docs/source/reference/image_classification.rst

```diff
@@ -55,6 +55,17 @@ Here's the full example:
     :language: python
     :lines: 14-
 
+Flash automatically takes care of Image transformations for you. But sometimes it is not enough and you want to customize the transformations.
+You can easily customize `Preprocessing <https://github.com/PyTorchLightning/lightning-flash/blob/master/flash/core/data/process.py>`_ method in the `data_module`.
+The base `Preprocess` class defines 7 methods for different stages in the data loading pipeline.
+To apply image augmentations you can directly use `default_transforms` defined in Flash Image module and then add your merge your custom defined image transformations with it.
+In this example, we will load the default transforms and merge it with custom `torchvision` transformations. We use `post_tensor_transform` since the transformations will be applied
+after image is converted into `torch.Tensor`.
+
+.. literalinclude:: ../../../flash_examples/image_transforms.py
+    :language: python
```

Comment:
Let's use testcode here and have the example directly in the docs.

Reply:
didn't get what you mean by "Let's use testcode here" 😅. Could you pls explain this?